pkg/integrations/kafka_lag_exporter/kafka_lag_exporter.go

```diff
@@ -0,0 +1,173 @@
+package kafka_lag_exporter
+
+import (
+	"fmt"
+
+	"github.com/go-kit/kit/log"
+	"github.com/grafana/agent/pkg/integrations"
+	"github.com/grafana/agent/pkg/integrations/config"
+
+	"github.com/Shopify/sarama"
+	kafka_exporter "github.com/davidmparrott/kafka_exporter/v2/exporter"
+)
+
+// DefaultConfig holds the default settings for the elasticsearch_exporter
+// integration.
+var DefaultConfig = Config{
+	KafkaUri:                []string{"localhost:11091"},
+	UseSASL:                 true,
+	UseTLS:                  false,
+	SaslUsername:            "test",
+	SaslPassword:            "test",
+	KafkaVersion:            sarama.V2_0_0_0.String(),
+	UseZooKeeperLag:         false,
+	Labels:                  "dev-cluster",
+	MetadataRefreshInterval: "1m",
+	AllowConcurrent:         true,
+	MaxOffsets:              1000,
+	PruneIntervalSeconds:    30,
+}
+
+type Config struct {
```

Comment:
Can you comment this type with a godoc comment? `// Config controls kafka_exporter`, or something like that.